performance/tests/Button.test.js

```diff
@@ -1,82 +1,139 @@
+		});
+	});
+
+	it('should have a good FID and CLS', async () => {
+		await page.evaluateOnNewDocument(FID);
+		await page.evaluateOnNewDocument(CLS);
+		await page.goto('http://localhost:8080/button');
+		await page.waitForSelector('#button');
+		await page.focus('#button');
+		await page.keyboard.down('Enter');
+		await page.waitForTimeout(200);
 
-			const actualFPS = FPS(filename);
-			TestResults.addResult({component: 'Button', type: 'Frames Per Second', actualValue: actualFPS});
+		let actualFirstInput = await page.evaluate(() => {
+			return window.fid;
+		});
 
-			const actualUpdateTime = Update(filename, 'Touchable');
-			TestResults.addResult({component: 'Button', type: 'Update', actualValue: actualUpdateTime});
+		let actualCLS = await page.evaluate(() => {
+			return window.cls;
 		});
+
+		TestResults.addResult({component: component, type: 'First Input Delay', actualValue: actualFirstInput});
+		TestResults.addResult({component: component, type: 'CLS', actualValue: actualCLS});
+
+		expect(actualFirstInput).toBeLessThan(maxFID);
+		expect(actualCLS).toBeLessThan(maxCLS);
 	});
 
-	it('should mount Button under threshold', async () => {
-		const filename = getFileName('Button');
+	it('should have a good DCL, FCP and LCP', async () => {
+		const filename = getFileName(component);
 
-		await page.tracing.start({path: filename, screenshots: false});
-		await page.goto('http://localhost:8080/button');
-		await page.waitForSelector('#button');
+		let contDCL = 0;
+		let contFCP = 0;
+		let contLCP = 0;
+		let avgDCL = 0;
+		let avgFCP = 0;
+		let avgLCP = 0;
+		for (let step = 0; step < stepNumber; step++) {
+			const page = await testMultiple.newPage();
+
+			await page.tracing.start({path: filename, screenshots: false});
+			await page.goto('http://localhost:8080/button');
+			await page.waitForSelector('#button');
+			await page.waitForTimeout(200);
+
+			await page.tracing.stop();
+
+			const actualDCL = await DCL(filename);
+			avgDCL = avgDCL + actualDCL;
+			if (actualDCL < maxDCL) {
+				contDCL += 1;
+			}
+
+			const actualFCP = await FCP(filename);
+			avgFCP = avgFCP + actualFCP;
+			if (actualFCP < maxFCP) {
+				contFCP += 1;
+			}
+
+			const actualLCP = await LCP(filename);
+			avgLCP = avgLCP + actualLCP;
+			if (actualLCP < maxLCP) {
+				contLCP += 1;
+			}
+
+			await page.close();
+		}
+		avgDCL = avgDCL / stepNumber;
+		avgFCP = avgFCP / stepNumber;
+		avgLCP = avgLCP / stepNumber;
+
+		TestResults.addResult({component: component, type: 'average DCL', actualValue: avgDCL});
+		TestResults.addResult({component: component, type: 'average FCP', actualValue: avgFCP});
+		TestResults.addResult({component: component, type: 'average LCP', actualValue: avgLCP});
+
+		expect(contDCL).toBeGreaterThan(percent);
+		expect(avgDCL).toBeLessThan(maxDCL);
 
-		await page.tracing.stop();
+		expect(contFCP).toBeGreaterThan(percent);
```

Comment:
7 is not percent.
How about 
global.passRatio = 0.7;
		expect(contDCL).toBeGreaterThan(passRatio*stepNumber);